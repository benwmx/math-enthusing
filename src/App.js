import React from 'react';
import './App.css';
import Calculator from './components/Calculator';

function App() {
  return (

    <div className="App">
      <header className="App-header">
        <p>
          <Calculator />
        </p>
      </header>
    </div>
  );
}

export default App;
