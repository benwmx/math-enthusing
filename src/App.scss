@import url('https://fonts.googleapis.com/css2?family=Dancing+Script:wght@400;500;600;700&display=swap');

$indigo: #432f70;
$purpleHaze:#713770;
$fuchsia: #b34270;
$roseRed: #e95670;
$blueGreen: #00b1b0;
$freesia: #fec84d;
$navyBlue: #284b70;
$bgMain: #8ad6cf;
$fontScript: 'Dancing Script', cursive;

:root {
  margin: 0;
  padding: 0;
  height: 100%;
}

body {
  width: 100%;
  height: 100%;
}

#root {
  width: 100%;
  height: 100%;
  background-color: $bgMain;
}

.navbar {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 10px 50px;
  height: 80px;
  background-color: $navyBlue;

  h1 {
    font-family: $fontScript;
    font-weight: bold;
    color: $blueGreen;
    font-size: 2.5rem;
    margin: 0;
  }

  .nav-links {
    width: 300px;
    display: flex;
    justify-content: space-between;
    padding-left: 0;

    a {
      font-style: italic;
      text-decoration: none;
      font-size: 1.1rem;
      color: $freesia;
    }

    .active-link {
      font-style: normal;
      font-weight: bold;
      color: $blueGreen;
    }
  }
}

.container {
  width: 90%;
  margin: 50px auto;
  color: $indigo;
}

.title {
  margin: 0;
  font-size: 2rem;
}

.container-calc {
  display: flex;
  justify-content: space-between;
}

.home-details {
  word-spacing: 5px;
  font-size: 1.3rem;
  margin-top: 70px;
}

.quote {
  @extend .home-details;

  margin-top: 100px;
}
