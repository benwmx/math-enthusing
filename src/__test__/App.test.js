import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

describe('App Component test', () => {
  it('Should render', () => {
    const app = renderer.create(<App />).toJSON();
    expect(app).toMatchSnapshot();
  });
});
