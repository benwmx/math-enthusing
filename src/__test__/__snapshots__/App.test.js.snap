// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component test Should render 1`] = `
Array [
  <header
    className="navbar"
  >
    <h1>
      Math Enthusing
    </h1>
    <ul
      className="nav-links"
    >
      <a
        aria-current="page"
        className="active-link"
        href="#/"
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
      <a
        aria-current={null}
        href="#/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        aria-current={null}
        href="#/Quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </ul>
  </header>,
  <div
    className="home container"
  >
    <h2
      className="title"
    >
      Welcome to our Page
    </h2>
    <p
      className="home-details"
    >
      lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor incidid lorem inc lorem ipsum dolor sit amet, consectetur adiplore, sed do eius mod lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor incididunt lorem inc lore sit amet et alice. Lore m lorem ips lorem  ipsum dolor sit amet,  consectet lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor incidid lorem inc lorem ipsum dolor sit amet, consectetur adiplore, sed do eius mod lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor incididunt lorem inc lore sit amet et alice. Lore m lorem ips lorem  ipsum dolor sit amet,  consectet lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor incidid lorem inc lorem ipsum dolor sit amet, consectetur adiplore, sed do eius mod lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor incididunt lorem inc lore sit amet et alice. Lore m lorem ips lorem  ipsum dolor sit amet,  consectet lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor
    </p>
  </div>,
]
`;
